### PR TITLE
Add blocktest to the robotology-superbuild

### PR DESCRIPTION
**Pull request by Nicogene**
This is a first draft for adding `blocktest` and `blocktest-yarp-plugins` to the `robotology-superbuild`.

For sure it misses:

- [x] CI
- [x] Documentation

### CODE REVIEW

**Comment by traversaro** [comment]
I would strongly prefer to avoid having single dependency components, as it increase complexity for CI, documentation and  in general user support. As `blocktest`  is related to robot testing, do you think it could make sense to add it to the existing `ROBOTOLOGY_ENABLE_ROBOT_TESTING` component? To clarify, the `ROBOT_TESTING` in the component name is a generic term, it is not relative specifically to the `robot-testing-framework` .

**Comment by Nicogene** [comment]
For me is fine, I was worried only for the ambiguity of the name and the differences of dependecies of the 2 repository. My main concern is about boost, it would require to install boost if you want to compile `robot-testing-framework` via superbuild am I right?
I totally agree about the reduction of complexity

**Comment by traversaro** [comment]
>  My main concern is about boost, it would require to install boost if you want to compile robot-testing-framework via superbuild am I right? I totally agree about the reduction of complexity

I think the problem created by this downside is minor w.r.t. to the advantages by having just one component.

**Comment by traversaro** [comment]
In the meanwhile, I think if one of these two repos have an unstable (`devel` ) branch, you can add it in `cmake/ProjectTagsUnstable.cmake` .

**Comment by Nicogene** [comment]
I added it under the ROBOT_TESTING profile, in this way both documentation and CI should be ok, am I right?

**Comment by traversaro** [comment]
> 
> 
> I added it under the ROBOT_TESTING profile, in this way both documentation and CI should be ok, am I right?

Ok for the CI, for the docs probably you may want to update the table in https://github.com/robotology/robotology-superbuild#profile-cmake-options  ?

File: README.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ The steps necessary to install the system dependencies of the Robot Testing prof
 operating system-specific installation documentation, and no additional required system dependency is required.
 
 ### Check the installation
-If the profile has been correctly enabled and compiler, you should be able to run the `robottestingframework-testrunner` executable from the command line.
+If the profile has been correctly enabled and compiler, you should be able to run the `robottestingframework-testrunner` and `blocktestrunner`executable from the command line.
```
Comment (traversaro) [inline README.md:389]:
```suggestion
If the profile has been correctly enabled and compiler, you should be able to run the `robottestingframework-testrunner` and `blocktestrunner` executable from the command line.
```

**Comment by traversaro** [comment]
CI passed at the first trial, nice!

**Comment by traversaro** [comment]
Thanks  @Nicogene !

**Pull request merged**